test/functions.test.js

```diff
@@ -1,5 +1,6 @@
 // IMPORT MODULES under test here:
 import { 
+    getSecondItem,
     myFunction,
```

Comment:
Delete this function and its corresponding test, it doesn't exist and is just for syntax reference!